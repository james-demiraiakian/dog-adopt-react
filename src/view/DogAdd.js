import React, { useEffect, useState } from 'react';
import { useHistory } from 'react-router-dom';
import { useParams } from 'react-router-dom';

export default function DogAdd() {
  const [dog, setDog] = useState({});

  const params = useParams();
  const history = useHistory();

  return <div></div>;
}
