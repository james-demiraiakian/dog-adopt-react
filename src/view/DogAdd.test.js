import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import DogAdd from './DogAdd';

test('should render an empty test page', () => {
  const { container } = render(
    <MemoryRouter>
      <DogAdd />
    </MemoryRouter>
  );

  screen.findByPlaceholderText('Name');

  expect(container).toMatchSnapshot();
});
