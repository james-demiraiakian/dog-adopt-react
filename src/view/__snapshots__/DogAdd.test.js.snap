// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an empty test page 1`] = `
<div>
  <div>
    <h2>
      Add Dog
    </h2>
    <div>
      <form
        class="dog-form"
      >
        <input
          placeholder="Name"
          type="text"
          value=""
        />
        <input
          placeholder="Age"
          type="number"
          value="0"
        />
        <input
          placeholder="Breed"
          type="text"
          value=""
        />
        <input
          placeholder="Bio"
          type="text"
          value=""
        />
        <input
          placeholder="Image URL"
          type="text"
          value=""
        />
        <button>
          Submit
        </button>
      </form>
    </div>
    <h1 />
  </div>
</div>
`;
